src/alire/alire-actions.adb

```diff
@@ -17,7 +17,7 @@ package body Alire.Actions is
    -- To_TOML --
    -------------
 
-   function To_TOML (This : Run) return TOML.TOML_Value is
+   overriding function To_TOML (This : Run) return TOML.TOML_Value is
```

Comment:
I always put `overriding` on the line above `function`.

Reply:
OK, will do in the future, thanks.